docs/ModuleWidgets.md

```diff
@@ -0,0 +1,39 @@
+## Module Widgets
+
+Since LORIS 23.0, LORIS incorporates a concept of "widgets" which
+modules can use to dynamically display graphical widgets in other
+modules when installed and activated. This is a powerful way to
+extend the capabilities of other modules within a new module.
+
+Widgets are can be used by implementing the `getWidgets` method
+on a `Module` module descriptor class. The first parameter of
+`getWidgets` is a type string, allowing modules to define their
```

Comment:
I'm not sure I understand the phrase "type string"

Reply:
The parameter named `$type` which is a string.

Reply:
```suggestion
`getWidgets` is a string representing the Widget type, allowing modules to define their
```